examples/bri-3/src/bri/communication/api/messages.controller.spec.ts

```diff
@@ -191,6 +179,12 @@ describe('MessageController', () => {
         signature: 'xyz',
         type: 1,
       } as CreateBpiMessageDto;
+      subjectStorageAgentMock.getBpiSubjectById.mockResolvedValueOnce(
+        undefined,
+      );
+      subjectStorageAgentMock.getBpiSubjectById.mockResolvedValueOnce(
```

Comment:
can you please point me to the difference between this and the line above?

Reply:
no difference, just 2 calls during test execution, we mock both

Reply:
Is there a method in the mock package that could be called to setup the mocking behaviour once? i.e. mockResolvedValueAlways? Might be less confusing. Otherwise ok.

Reply:
usually test packages don't need to mock two calls twice. There must be a way to handle this